app/models/git/exercise.rb

```diff
@@ -52,12 +62,22 @@ def example
 
     memoize
     def authors
-      config[:authors]
+      config[:authors] || []
```

Comment:
This makes it slightly easier to work with.

Reply:
```suggestion
      config[:authors].to_a
```